### PR TITLE
fix: remove tab trap (DE44729)

### PR DESCRIPTION
**Pull request by devpow112**
Also fixes some styling issues that cause jittering.

Rally: [DE44729](https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fdefect%2F606230002809)

### CODE REVIEW

File: d2l-rubric-editable-score.js [hunk 1]
```diff
@@ -350,7 +349,7 @@ Polymer({
 	],
 
 	_handleInteraction: function(e) {
-		if (e.eventType === 'keydown' && e.keyCode !== ENTER_KEY) {
+		if (e.type === 'keydown' && e.keyCode !== ENTER_KEY) {
```
Comment (devpow112) [inline d2l-rubric-editable-score.js:352]: Can't believe it took me so long to figure this out 😢.

File: d2l-rubric-editable-score.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-rubric-editable-score">
 				margin-left: 0.25rem;
 				margin-right: 0.5rem;
 				display: inline-flex;
-				line-height: 2rem;
+				line-height: 2.25rem;
```
Comment (devpow112) [inline d2l-rubric-editable-score.js:72]: Stop some jitter in the UI when switching between input box being shown and not.

Comment (semibran) [reply d2l-rubric-editable-score.js:72]: This input box is just gonna keep getting taller

**Review by semibran — APPROVED** [review]

File: d2l-rubric-editable-score.js (hunk 2 above)
Comment (devpow112) [reply d2l-rubric-editable-score.js:72]: Lol yup 😢

**Pull request merged**